public/app/features/alerting/unified/components/rules/central-state-history/EventListSceneObject.tsx

```diff
@@ -166,40 +212,52 @@ function EventRow({ record, logRecords }: EventRowProps) {
 interface AlertRuleNameProps {
   labels: Record<string, string>;
   ruleUID?: string;
+  addFilterByName: (alertRuleName: string) => void;
 }
-function AlertRuleName({ labels, ruleUID }: AlertRuleNameProps) {
+function AlertRuleName({ labels, ruleUID, addFilterByName }: AlertRuleNameProps) {
   const styles = useStyles2(getStyles);
   const alertRuleName = labels['alertname'];
   if (!ruleUID) {
     return (
       <Text>
         <Trans i18nKey="alerting.central-alert-history.details.unknown-rule">Unknown</Trans>
-        {alertRuleName}
       </Text>
     );
   }
+  const ariaLabel = t('central-alert-history.details.add-filter', 'Add filter by alert name');
   return (
-    <Tooltip content={alertRuleName ?? ''}>
-      <a
-        href={`/alerting/${GRAFANA_RULES_SOURCE_NAME}/${ruleUID}/view?returnTo=${encodeURIComponent('/alerting/history')}`}
-        className={styles.alertName}
-      >
-        {alertRuleName}
-      </a>
-    </Tooltip>
+    <Stack gap={1} direction={'row'} alignItems="center">
+      <Tooltip content={alertRuleName ?? ''}>
+        <a
+          href={`/alerting/${GRAFANA_RULES_SOURCE_NAME}/${ruleUID}/view?returnTo=${encodeURIComponent('/alerting/history')}`}
+          className={styles.alertName}
+        >
+          {alertRuleName}
+        </a>
+      </Tooltip>
+      <IconButton
```

Comment:
in this case clicking the alert name is not an option to add the filter, as it's an anchor to the rule view page.